### DISCUSSION TITLE
Need a contributor who can help me out organizing the table.

### DISCUSSION THREAD

**Discussion by vvxxrrvvx** [community/community#61567] [Other Feature Feedback, Questions, & Ideas]
### repository : https://github.com/vvxxrrvvx/codewars-kata

---

### Example: 

<p align="center"> 
<img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/python/python-original.svg" width="85" height="90"/>
</p>

# 1 kyu

<br>

| Solution | Kata |
|--|--|
| [BECOME IMMORTAL](https://github.com/vvxxrrvvx/codewars-kata/blob/main/Python/1%20kyu/1Kyu_BECOME_IMMORTAL.py) | [link](https://www.codewars.com/kata/59568be9cc15b57637000054) |
| [Faberge easter eggs crush test [linear]](https://github.com/vvxxrrvvx/codewars-kata/blob/main/Python/1%20kyu/1kyu_Faberge%20easter%20eggs%20crush%20test%20%5Blinear%5D.py) | [link](https://www.codewars.com/kata/5976c5a5cd933a7bbd000029) |

<br>

**Comment by GhostOf0days** [comment]
You should also add a languages column.
